packages/editor/src/components/post-featured-image/index.js

```diff
@@ -24,22 +24,25 @@ const ALLOWED_MEDIA_TYPES = [ 'image' ];
 const DEFAULT_FEATURE_IMAGE_LABEL = __( 'Featured Image' );
 const DEFAULT_SET_FEATURE_IMAGE_LABEL = __( 'Set featured image' );
 const DEFAULT_REMOVE_FEATURE_IMAGE_LABEL = __( 'Remove image' );
+const path = require( 'path' );
```

Comment:
`path` is Node based and won't work as expected in the browser. You won't see it in the `node_modules` folder.